python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -988,60 +853,54 @@ interface {{ ethernet_interface.name }}
 {%     elif ethernet_interface.spanning_tree_bpdufilter is arista.avd.defined("disabled") %}
    spanning-tree bpdufilter disable
 {%     endif %}
+{%     if ethernet_interface.logging.event.spanning_tree is arista.avd.defined(true) %}
+   logging event spanning-tree
+{%     elif ethernet_interface.logging.event.spanning_tree is arista.avd.defined(false) %}
+   no logging event spanning-tree
+{%     endif %}
 {%     if ethernet_interface.spanning_tree_guard is arista.avd.defined %}
 {%         if ethernet_interface.spanning_tree_guard == 'disabled' %}
    spanning-tree guard none
```

Comment:
this will come before to logging event spanning-tree

Reply:
weird. Fine to keep all of them where the root and none is.